xml/System/ReadOnlySpan`1.xml

```diff
@@ -483,11 +531,14 @@
         <Parameter Name="right" Type="System.ReadOnlySpan&lt;T&gt;" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="left">The first span to compare.</param>
+        <param name="right">The second span to compare.</param>
+        <summary>Returns a value that indicates whether two <see cref="T:System.ReadOnlySpan`1" /> objects are equal.</summary>
```

Comment:
> "object"... is that the nomenclature we use even for value types?

I think @stephentoub's previous comment applies here (and elsewhere) within the doc as well.

Reply:
What do I use?

Reply:
@BillWagner

Reply:
If you know type specific type, I'd use the specific type (e.g. `int`, `ReadOnlySpan<T>`, etc. If it must be a value type (but not a specific type), use the phrase "value type".